### DISCUSSION TITLE
customprefix doesnt work

### DISCUSSION THREAD

**Discussion by legendskyk** [inspircd/inspircd#2059] [Support]
```
<module name="customprefix">
<customprefix name="founder"
              letter="q"
              prefix="~"
              rank="50000"
              ranktoset="50000"
              ranktounset="50000"
              depriv="yes">

<customprefix name="admin"
              letter="a"
              prefix="&amp;"
              rank="40000"
              ranktoset="50000"
              ranktounset="50000"
              depriv="yes">

<customprefix name="halfop"
              letter="h"
              prefix="%"
              rank="20000"
              ranktoset="30000"
              ranktounset="30000"
              depriv="yes">		

<customprefix name="voice"
              change="yes"
              depriv="yes">		
```	

I did set this but we are not get prefix mode in room founder mode +q etc... how does it work or enable it in new version?

**Comment by sadiepowell** [comment]
If you want users to receive `+q` on creating a channel you need to add it to `<options:defaultmodes>` and rehash.

https://docs.inspircd.org/3/configuration/#options

**Reply by legendskyk** [reply]
![image](https://github.com/inspircd/inspircd/assets/42233490/9ca15f18-9aa9-4138-979e-fa0c716ceec1)
i did that still its not setting prefix on nicklist i attack image see it

**Reply by sadiepowell** [reply]
If its not showing the prefix then you probably connected after the mode was added so you'll need reconnect for it to show properly. Also make sure your client is up to date as some older clients have issues showing this properly.

**Reply by sadiepowell** [reply]
Did that resolve your problem?
